coalib/settings/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
+Has Setting, Section class; indexing tools; input checks; loads bears, files.
+
+- **class Setting:** Aggregates the settings passed to coala.
+- **class Section:** Parses the Settings given in each section in config file.
+- **class FunctionMetadata:** Provides indexing tools from python docstrings.
```

Comment:
this is not correct.  This has all the stuff needed to make bear arg annotation work